containers/ecr-viewer/src/app/api/services/dialects/sqlserver.ts

```diff
@@ -1,6 +1,9 @@
 import { MssqlDialect } from "kysely";
 import * as tarn from "tarn";
 import * as tedious from "tedious";
+import parse from "mssql-connection-string";
+
+const dbConfig = parse(process.env.DATABASE_URL || "");
```

Comment:
```suggestion
const { password, user, database, port, server, ...others } = parse(process.env.DATABASE_URL || "");
```

is this actually valid js? not sure, but I wonder if there's another place to shove extra config? @BobanL do you know about this oen?

Reply:
This library in general looks like to return a very small subset of possible attributes in that url. 

https://github.com/nsc-c-park/mssql-connection-string/blob/master/src/mssql-connection-string.js#L23C9 
vs
https://github.com/tediousjs/node-mssql/blob/master/lib/base/connection-pool.js#L124

Reply:
There are nearly 100 values we would need to support if we wanted to make a database URL work for MSSQL. Specifically, this is not an expectation that is supported by the SQL Server community. Contrast that with postgres and MySQL, which have, since their outsets, explicitly had the goal of supporting URL-based connection strings. 

I suggest that for now, in order to get this unblocked, we only support the attributes that are already supported in main. There _is not_ and off-the-shelf TypeScript library that supports connecting to MS SQL with an arbitrary URL. 

@mcmcgrath13 and @BobanL, If you have a specific fix you'd like to suggest, I'm open to it. As a matter of completing this task, I suggest we move forward with what's already there. Thoughts?

Reply:
Looking at the library internals, I don't see any way to pass through the connection string as is. @BobanL do you know which attributes users needed?

Reply:
switching to the tedious mssql connection parser seems like a fine idea either way, but doesn't really solve us as being in the middle of mapping the parsed outputs to reasonable places

Reply:
@BobanL @akasper I went ahead and switched over to the `mssql` connection parser and shoved everything I could find in a place that seemed like where it could go. I'm sure we'll find gaps in the future, but I vote we deal with it when it becomes a problem

Reply:
That sounds like the best idea!